README.md

```diff
@@ -9,38 +9,60 @@ A practical functional library for Javascript programmers.
 [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/ramda/ramda?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 
-Goals
------
+Why Ramda?
+----------
 
 <img src="http://ramda.jcphillipps.com/logo/ramdaFilled_200x235.png" 
-     width="200" height="235" align="left" hspace="12" />
-Using this library should feel as much like using Javascript as 
-possible.  Of course it's functional Javascript, but we're not 
-introducing lambda expressions in strings, we're not borrowing consed 
+     width="170" height="190" align="right" hspace="12" />
+
+Many excellent libraries with functional flavor,
+such as _Underscore_ and _Lodash_ already exist. So why did we create Ramda?
+We need some high level philosphical reasons why you guys created ramda instead of using underscore and such.
+lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem lorem 
+
```

Comment:
How about this?: 

> There are already several excellent libraries with a functional flavor such as _Underscore_ and _Lo-Dash_.  But those are meant to be general-purpose toolkits, suitable for working in multiple paradigms.  We wanted a library designed specifically for a functional programming style, one that made it easy to create functional pipelines, one that never mutated user data. 

BTW, note the spelling and capitalization of Lo-Dash.  JDD has taken me to task for this once.  I'd rather not get it wrong in our ReadMe.  :smiling_imp: